docs/Getting-Started.md

```diff
@@ -67,6 +67,8 @@ Fastify offers an easy platform that helps solve all of problems, and more.
 > })
 > ```
 >
+> Similarly, specify `::1` to accept only local connections via IPv6. Or specify `::` to accept connections on all IPv6 addresses, and, if the operating system supports it, also on all IPv4 addresses.
```

Comment:
@jsumners Maybe here we can link the nodejs doc?
https://nodejs.org/docs/latest/api/net.html#net_server_listen_port_host_backlog_callback

Reply:
/shrug

I think the text covers it. People can do some research if they want to know more.